### ISSUE TITLE
Can't install rio 0.6 on k8s 1.16.2  - 100% CPU load

### ISSUE THREAD

**Issue by Vad1mo** [rancher/rio#605]
I tried to install rio 0.6 alpah on k8s 1.16.2

It`s a fresh vanilla k8s.

On my Mac the command `rio install --http-port 80 --https-port 443` hangs. 
The CPU load rises to 100% but no resources are created on k8s, not even the namespace is created after minutes. 

`rio version v0.6.0-alpha.1 (7678d67d)`

**Comment by Vad1mo** [comment]
I just tried the command `rio install` and it seems to work.


```
Deploying Rio control plane....
\ Warning: ClusterDomain is not accessible. Error: Get http://bbk62b.on-rio.io:80: dial tcp: lookup bbk62b.on-rio.io on 192.168.0.1:53: no such host
```

**Comment by izaac** [comment]
this is related to or might be a dupe of: https://github.com/rancher/rio/issues/599

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha2.

**Comment by rancher-max** [comment]
This should work now in `v0.6.0-alpha.2`. By default it does `--http-port 80` and `--https-port 443`, so you should just be able to run `rio install` in your cluster using the latest alpha version of rio.
